src/BackgroundTask/MonitorCssTransientCaching.php

```diff
@@ -65,20 +64,13 @@ final class MonitorCssTransientCaching extends RecurringBackgroundTask {
 	 */
 	const GUTENBERG_VERSION = 'gutenberg_version';
 
-	/**
-	 * @var BlockUniqidTransformer
-	 */
-	private $block_uniqid_transformer;
-
 	/**
 	 * Constructor.
 	 *
 	 * @param BackgroundTaskDeactivator $background_task_deactivator Deactivator.
-	 * @param BlockUniqidTransformer    $block_uniqid_transformer    Transformer.
 	 */
-	public function __construct( BackgroundTaskDeactivator $background_task_deactivator, BlockUniqidTransformer $block_uniqid_transformer ) {
+	public function __construct( BackgroundTaskDeactivator $background_task_deactivator ) { // phpcs:ignore Generic.CodeAnalysis.UselessOverridingMethod.Found
```

Comment:
Should the constructor just be removed instead? Then no need for the `phpcs:ignore` comment, right?

Reply:
We still need to pass `BackgroundTaskDeactivator` instance to the parent constructor... right?

Reply:
I don't think so because the parent class's `__constructor()` will be inherited by this subclass, or rather this class's grandparent class, `CronBasedBackgroundTask`:

https://github.com/ampproject/amp-wp/blob/22012c53f1015ae2496276c9fe2e7d09164d163b/src/BackgroundTask/CronBasedBackgroundTask.php#L38-L40

Reply:
Humm yes. What confused me was the DI autowiring in abstract and concrete classes. I have removed this override.